frontend/src/app/commonComponents/FormGroup.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./FormGroup.scss";
+
+interface Props {
+  title: string;
+  children: React.ReactNode;
+}
+
+const FormGroup = (props: Props) => (
```

Comment:
Previously this was being done with a native `<fieldset/>` but that ends up nesting `<fieldset/>` elements which supposedly confuses many assistive technologies. Since these sections already have headings (the `<h3>` here) it seems better to just make them visually stand out because screen readers will let people navigate by the heading hierarchy.